types/stripe/index.d.ts

```diff
@@ -5,11 +5,12 @@
 //                 Amrit Kahlon <https://github.com/amritk/>
 //                 Adam Cmiel <https://github.com/adamcmiel>
 //                 Justin Leider <https://github.com/jleider>
+//                 Kamil Gałuszka <https://github.com/galuszkak>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
```

Comment:
I didn't know it's possible to have two versions alongside but that does make sense.

I don't know if it is possible to have something extended optionally in TypeScript. I think we should look into possibility of some kind of extension rather than export.